docs/source/release_notes/release-notes.rst

```diff
@@ -63,6 +63,17 @@ develop
            Previously, clients would immediately retry the connection on the node with a 503 two times (for a total of three attempts) before failing over.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1782>`__)
 
+    *    - |improved| |devbreak|
+         - The format of exception messages has been brought in line with that of the `palantir/http-remoting <https://github.com/palantir/http-remoting>`__ library.
```

Comment:
Sure, that's right. I should have specified that this has to do with remote exceptions.